DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_forward_block_different_packets.py

```diff
@@ -0,0 +1,212 @@
+    """
+    Test Name: test_bridging_forward_block_different_packets
+    Test Suite: suite_functional_bridging
+    Test Overview: Verify that bridge forwarding/drop of different IPv4 packet types.
+    Test Author: Kostiantyn Stavruk
+    Test Procedure:
+    1. Init bridge entity br0.
+    2. Set ports swp1, swp2, swp3, swp4 master br0.
+    3. Set entities swp1, swp2, swp3, swp4 UP state.
+    4. Set bridge br0 admin state UP.
+    5. Send different types of packets from source TG.
+    6. Analyze traffic behaviour taking into account the traffic type that was sent.
+    """
+
+    bridge = "br0"
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 2)
+    if not tgen_dev or not dent_devices:
+        print("The testbed does not have enough dent with tgen connections")
+        return
+    dent_dev = dent_devices[0]
+    device_host_name = dent_dev.host_name
+    tg_ports = tgen_dev.links_dict[device_host_name][0]
+    ports = tgen_dev.links_dict[device_host_name][1]
+    traffic_duration = 10
+    srcMac = "00:00:AA:00:00:01"
+
+    out = await IpLink.add(
+        input_data=[{device_host_name: [
+            {"device": bridge, "type": "bridge"}]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Verify that bridge created.\n{out}"
+
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": bridge, "operstate": "up"}]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Verify that bridge set to 'UP' state.\n{out}"
+
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": port, "master": bridge, "operstate": "up"} for port in ports]}])
+    err_msg = f"Verify that bridge entities set to 'UP' state and links enslaved to bridge.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+
+    address_map = (
+        # swp port, tg port,    tg ip,     gw,        plen
+        (ports[0], tg_ports[0], "1.1.1.2", "1.1.1.1", 24),
+        (ports[1], tg_ports[1], "1.1.1.3", "1.1.1.1", 24)
+    )
+
+    dev_groups = tgen_utils_dev_groups_from_config(
+        {"ixp": port, "ip": ip, "gw": gw, "plen": plen}
+        for _, port, ip, gw, plen in address_map
+    )
+
+    await tgen_utils_traffic_generator_connect(tgen_dev, tg_ports, ports, dev_groups)
+
+    streams = {
+        "IPv4_Broadcast": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
+            "srcMac": srcMac,
+            "dstMac": "FF:FF:FF:FF:FF:FF",
+            "frameSize": 96,
+            "protocol": "ip",
+            "type" :"raw"
+        },
+        "IPv4_ReservedMC": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcIp": "1.1.1.2",
+            "dstIp": "1.1.1.3",
```

Comment:
I have a doubt that the packet is correct.
Common comment:
After you have configured a stream, go to the IxNetwork GUI and check it. If the stream is invalid, GUI may give you a hit about what is not correct.